src/python_testing/TC_MCORE_FS_1_2.py

```diff
@@ -0,0 +1,185 @@
+# for details about the block below.
+#
+
+import ipaddress
+import logging
+import os
+import pathlib
+import queue
+import random
+import signal
+import subprocess
+import time
+import uuid
+
+import chip.clusters as Clusters
+from chip import ChipDeviceCtrl
+from chip.interaction_model import InteractionModelError, Status
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main, has_cluster, per_endpoint_test
+from mobly import asserts
+from TC_SC_3_6 import AttributeChangeAccumulator
+
+
+class TC_MCORE_FS_1_2(MatterBaseTest):
+
+    @async_test_body
+    async def setup_class(self):
+        super().setup_class()
+
+        # Create a second controller on a new fabric to act as the TH_FSA and commission devices through the DUT_FSA
+        new_certificate_authority = self.certificate_authority_manager.NewCertificateAuthority()
+        new_fabric_admin = new_certificate_authority.NewFabricAdmin(vendorId=0xFFF1, fabricId=2)
+        paa_path = str(self.matter_test_config.paa_trust_store_path)
+        self.TH_FSA = new_fabric_admin.NewController(nodeId=112233, paaTrustStorePath=paa_path)
+
+    def teardown_class(self):
+        super().teardown_class()
+
+    def steps_TC_MCORE_FS_1_2(self) -> list[TestStep]:
+        steps = [TestStep(1, "TH_FSA subscribes to all the Bridged Device Basic Information clusters provided by DUT_FSA to identify the presence of a Bridged Node endpoint with a UniqueID matching the UniqueID provided by the BasicInformationCluster of the TH_SED_DUT."),
+                 TestStep(2, "TH_FSA initiates commissioning of TH_SED_DUT by sending the OpenCommissioningWindow command to the Administrator Commissioning Cluster on the endpoint with the uniqueID matching that of TH_SED_DUT."),
+                 TestStep(3, "TH_FSA completes commissioning of TH_SED_DUT using the Enhanced Commissioning Method."),
+                 TestStep(4, "Commission TH_SED_L onto DUT_FSA’s fabric using the manufacturer specified mechanism."),
+                 TestStep(5, "TH_FSA waits for subscription report from a the Bridged Device Basic Information clusters provided by DUT_FSA to identify the presence of a Bridged Node endpoint with a UniqueID matching the UniqueID provided by the BasicInformationCluster of the TH_SED_L."),
+                 TestStep(6, "TH_FSA initiates commissions of TH_SED_L by sending the OpenCommissioningWindow command to the Administrator Commissioning Cluster on the endpoint with the uniqueID matching that of TH_SED_L."),
+                 TestStep(7, "TH_FSA completes commissioning of TH_SED_L using the Enhanced Commissioning Method.")]
+        return steps
+
+    @async_test_body
+    async def test_TC_MCORE_FS_1_2(self):
+        self.is_ci = self.check_pics('PICS_SDK_CI_ONLY')
+        min_report_interval_sec = self.user_params.get("min_report_interval_sec", 0)
```

Comment:
what are these? They're not mentioned in the test plan.

Reply:
I think these are per spec, I grabbed them from RR.

Reply:
0 and 10*60 seem like icd values. This just adds knobs that probably don't need twiddling and makes the default test REALLY long. 

I think these are coupled in a way that's going to cause problems. The delay for adding the endpoint isn't actually related to the subscription ceiling in a meaningful way, is it?

This says to expect a report (either with data or an empty report) some time between 0s and 10 minutes and to wait for 20 minutes for the endpoint to appear. 

If you change the subscription ceiling, you also change the expectation for how long the endpoint can take to appear.

Reply:
Looking at other things with timeouts, looks like 10s is a pretty standard value, so I'll update the report window to 2s and the max delay to 10s.